packages/suite/src/views/dashboard/AssetsView/AssetCoinLogo.tsx

```diff
@@ -35,7 +26,7 @@ export const AssetCoinLogo = ({ symbol, assetsFiatBalances, index }: AssetCoinLo
         : undefined;
 
     return (
-        <LogoWrapper>
+        <Row alignItems="center" justifyContent="center">
```

Comment:
alignItems="center" is the default

Reply:
https://github.com/trezor/trezor-suite/pull/14996/commits/d1b9c30e115df36aba8ad13e537786aa33226844